app/src/components/governance/ProposalStatusBar.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <section class="status-bar">
+    <div v-if="status === `Deposit Period`">
+      <div class="top row">
+        <div class="time">Entered Deposit Period on {{ statusBeginTime }}</div>
```

Comment:
can be refactored? `Entered XXXX on ...`

Reply:
Done